src/common/localization/en-us.ts

```diff
@@ -33,6 +33,12 @@ export const english: IAppStrings = {
         restore: "Restore",
         close: "Close",
     },
+    statusBar: {
+        newVersionAvaliable: {
+            title: "New version avaliable",
+            message: "New version ${newVersion.version}-${newVersion.commit} is avaliable, you can upgrade from fott site."
+        }
+    },
```

Comment:
I don't think they user can get new version from FOTT site (what site?). We can put a link to GitHub page here.